app/views/course/groups/_group.html.slim

```diff
@@ -2,11 +2,10 @@
   th = format_inline_text(group.name)
   td = group.course_users.with_approved_state.count
   td
-    ul
-      - group.course_users.managers.each do |manager|
-        li
-          = link_to_course_user(manager) do
-            = format_inline_text(manager.name)
+    - group.group_users.manager.map(&:course_user).each do |manager|
```

Comment:
`manger` is a scope, in this case it will still generate N queries. we might need to use `select` instead.